pkg/kernel/kernel.go

```diff
@@ -21,16 +21,20 @@ const (
 	kernelIdle     = "idle"
 )
 
+var (
+	// ExecCounter is incremented each time we run user code in the notebook.
+	// It cannot be defined in Kernel, or it will be always 1.
```

Comment:
This is a little misleading without any context, maybe rephrase it to `It cannot be defined in Kernel struct`?

Reply:
btw, what's the concrete use case of exec counter? is it required by jupyter?

Reply:
Yeah, it is required by Jupyter